create-cycle-app/index.js

```diff
@@ -11,6 +11,7 @@ var argv = require('minimist')(process.argv.slice(2))
 var pathExists = require('path-exists')
 var request = require('request')
 var inquirer = require('inquirer')
+var githubUrlFromNpm = require('github-url-from-npm')
```

Comment:
Does it really need this dependency? Looking at [the code](https://github.com/hokaccha/github-url-from-npm/blob/master/index.js), it does only make a request to npm API (that we certainly could call), and delegates to [another one](https://github.com/tj/node-github-url-from-git/blob/master/index.js) to just format a url from github.

Besides this, it enforces flavor's authors to publish package on github AND npm in order to be discovered, which seems duplicated effort. (There is a problem in using just npm data?)